gui/src/components/dashboard/dashboard.vue

```diff
@@ -57,11 +58,29 @@ export default class Dashboard extends Vue {
   public alertTblRowHeight: number = 0;
   public alertTblRowHeightPx: string = "";
   public chartRowHeightPx: string = "";
+  public colNumber: number = 8;
+
+  public data() {
+    return {
+      constString: require("../../common/const-string.json")
```

Comment:
As mentioned earlier in our previous discussions we don't need data methods to declare the properties. You can directly declare them in Class area. Also you can import the constants file as below
import commonContants from "../../common/const-string.json";

Reply:
Removed  required and added import statements in all the files.